frontend/packages/text-editor/src/TextRow.tsx

```diff
@@ -109,29 +118,26 @@ export const TextRow = ({
           </TableCell>
         );
       })}
-      <TableCell>
-        <ButtonContainer>
+      <TableCell className={classes.cellContent}>
+        <ButtonContainer className={classes.textIdContainer}>
           {textIdEditOpen ? (
-            <div>
-              <LegacyTextField
-                aria-label={'tekst key edit'}
-                isValid={!keyError}
-                value={textIdValue}
-                type='text'
-                onBlur={handleTextIdBlur}
-                onChange={handleTextIdChange}
-              />
-              {keyError ? <ErrorMessage role='alertdialog'>{keyError}</ErrorMessage> : null}
-            </div>
+            <Textfield
+              value={textIdValue}
+              aria-label={'text key edit'}
```

Comment:
I see that this is not a code you have implemented, but should we translate the aria-label to Norwegian using the nb.json language file? 😊 Aria-labels should match the same language as the rest of the application.